app/templates/confirmation.html

```diff
@@ -1,5 +1,5 @@
 {% extends 'questionnaire.html' %}
 
-{% block page_title %}Confirmation{% endblock %}
+{% block page_title %}{{page_title}}{% endblock %}
```

Comment:
Not sure it's worth it. I think most if not all pages have different page_titles now. You may have put this comment before I refactored based on @LJBabbage review